app/assets/stylesheets/band_show.css.scss

```diff
@@ -0,0 +1,15 @@
+.band-header
+{
+  width: 80%;
+  margin: 3% auto;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.